Multisig/UI/Assets/AssetsViewModel.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  AssetsViewModel.swift
+//  Multisig
+//
+//  Created by Andrey Scherbovich on 13.05.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+struct TokenBalance: Identifiable {
+    var id: String {
+        return address
+    }
+    var imageURL: URL {
+        return URL(string: "https://gnosis-safe-token-logos.s3.amazonaws.com/\(address).png")!
```

Comment:
here too, `return` can be removed

Reply:
the icon for ETH is not working.

Reply:
this will be replaced soon, just a tmp solution